src/java/fr/paris/lutece/plugins/forms/web/admin/FormResponseJspBean.java

```diff
@@ -40,58 +43,67 @@
-	
-	// Templates
-	private static final String TEMPLATE_VIEW_STEP = "/admin/plugins/forms/step_view.html";
-	private static final String TEMPLATE_FORM_SUBMITTED = "/admin/plugins/forms/form_submitted_view.html";
-	private static final String TEMPLATE_VIEW_FORM_RESPONSE_SUMMARY = "/admin/plugins/forms/form_response_summary.html";
-	private static final String TEMPLATE_VIEW_ERROR_PAGE = "/admin/plugins/forms/error_view.html";
-
-	// Properties for page titles
+    private static final long serialVersionUID = 1L;
+
+    // Views
+    private static final String VIEW_STEP = "stepView";
+    private static final String VIEW_ERROR = "errorView";
+
+    // Actions
+    private static final String ACTION_SAVE_FORM_RESPONSE = "doSaveResponse";
+    private static final String ACTION_SAVE_STEP = "doSaveStep";
+    private static final String ACTION_RESET_BACKUP = "doResetBackup";
+    private static final String ACTION_SAVE_FOR_BACKUP = "doSaveForBackup";
+    private static final String ACTION_PREVIOUS_STEP = "doReturnStep";
+    private static final String ACTION_GO_TO_STEP = "doGoToStep";
+    private static final String ACTION_FORM_RESPONSE_SUMMARY = "formResponseSummary";
+    private static final String ACTION_SAVE_FORM_RESPONSE_SUMMARY = "doSaveResponseSummary";
+    private static final String ACTION_ADD_ITERATION = "addIteration";
+    private static final String ACTION_REMOVE_ITERATION = "removeIteration";
+
+
+
+    // Templates
+    private static final String TEMPLATE_VIEW_STEP = "/admin/plugins/forms/step_view.html";
+    private static final String TEMPLATE_FORM_SUBMITTED = "/admin/plugins/forms/form_submitted_view.html";
+    private static final String TEMPLATE_VIEW_FORM_RESPONSE_SUMMARY = "/admin/plugins/forms/form_response_summary.html";
+    private static final String TEMPLATE_VIEW_ERROR_PAGE = "/admin/plugins/forms/error_view.html";
+
+    // Properties for page titles
     private static final String PROPERTY_PAGE_TITLE_CREATE_FORM_RESPONSE = "forms.create_form_response.pageTitle";
     private static final String PROPERTY_PAGE_TITLE_ERROR_FORM_RESPONSE = "forms.error_form_response.pageTitle";
-    
+
     // Markers
     private static final String STEP_HTML_MARKER = "stepContent";
     private static final String MARK_FORM_TITLE = "formTitle";
     private static final String MARK_LIST_SUMMARY_STEP_DISPLAY = "list_summary_step_display";
-    
+
     // Other
     private Controller _controller = getClass( ).getAnnotation( Controller.class );
     private FormService _formService = SpringContextService.getBean( FormService.BEAN_NAME );
     private FormResponseManager _formResponseManager;
     private Step _currentStep;
     private StepDisplayTree _stepDisplayTree;
     private IBreadcrumb _breadcrumb;
-	
+
+    private static final String MESSAGE_WARNING_LOST_SESSION = "forms.warning.lost.session";
+
+    private boolean IsRequestComingFromAction = false;
```

Comment:
What is the purpose of this session variable? It is valorized in the methods you have created, but it is not used anywhere.

Reply:
indeed I deleted the variable